cypress/support/commands.js

```diff
@@ -22,4 +22,82 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('byPlaceholder', (placeholder) => {
+  cy.get(`[placeholder = "${placeholder}"`);
+});
+
+Cypress.Commands.add('addWorker', (user, count) => {
+  while (count > 0) {
+    cy.get('#addNewRecordButton')
+      .should('contain', 'Add')
+      .click();
+    cy.byPlaceholder('First Name').type(user.firstName);
+    cy.byPlaceholder('Last Name').type(user.lastName);
+    cy.byPlaceholder('name@example.com').type(user.email);
+    cy.byPlaceholder('Age').type(user.age);
+    cy.byPlaceholder('Salary').type(user.salary);
+    cy.byPlaceholder('Department').type(user.department);
+    cy.get('#submit').click();
+    count--;
+  }
+});
+
+Cypress.Commands.add('checkWorker', (user) => {
+  cy.get('.rt-tr')
+    .should('contain', user.firstName)
+    .should('contain', user.lastName)
+    .should('contain', user.email)
+    .should('contain', user.age)
+    .should('contain', user.salary)
+    .should('contain', user.department);
+});
+
+Cypress.Commands.add('changeWorker', (changedUser) => {
+  cy.get('#firstName').type('{selectAll}' + changedUser.changedName);
+  cy.get('#lastName').type('{selectAll}' + changedUser.changedLastName);
+  cy.get('#userEmail').type('{selectAll}' + changedUser.changedEmail);
+  cy.get('#age').type('{selectAll}' + changedUser.changedAge);
+  cy.get('#salary').type('{selectAll}' + changedUser.changedSalary);
+  cy.get('#department').type('{selectAll}' + changedUser.changed_department);
+  cy.get('#submit').click();
+});
+
+Cypress.Commands.add('changeWorkerCheck', (changedUser) => {
+  cy.byPlaceholder('Type to search')
+    .type(changedUser.changedName);
+  cy.byPlaceholder('Type to search')
+    .type('{selectAll}' + changedUser.changedLastName);
+  cy.byPlaceholder('Type to search')
+    .type('{selectAll}' + changedUser.changedEmail);
+  cy.byPlaceholder('Type to search')
+    .type('{selectAll}' + changedUser.changedAge);
+  cy.byPlaceholder('Type to search')
+    .type('{selectAll}' + changedUser.changedSalary);
+  cy.byPlaceholder('Type to search')
+    .type('{selectAll}' + changedUser.changed_department);
+
+  cy.get('.rt-tr')
+    .should('contain', changedUser.changedName)
+    .should('contain', changedUser.changedLastName)
+    .should('contain', changedUser.changedEmail)
+    .should('contain', changedUser.changedAge)
+    .should('contain', changedUser.changedSalary)
+    .should('contain', changedUser.changed_department);
+});
+
+Cypress.Commands.add('findByOptions', (user) => {
+  cy.byPlaceholder('Type to search')
```

Comment:
you have to add .should methods to assert that the table contains the searched values. For example search by first name -> assert that table contain the searched first name then clear the search field and search by the next column

Reply:
i've added this, ty.